cmake/projects/SDL2/hunter.cmake

```diff
@@ -149,6 +149,17 @@ hunter_add_version(
         3b8d0dfdbd1b062035a9432bff75a22ab439bcd1
 )
 
+hunter_add_version(
+        PACKAGE_NAME
+        SDL2
+        VERSION
+        "2.0.18"
+        URL
+        "https://github.com/libsdl-org/SDL/archive/release-2.0.18.tar.gz"
```

Comment:
SDL can now be used directly from upstream? I doubt SDL has no external dependencies for building. Or did they themself add Hunter support?

Reply:
Yes. It has no external dependencies. 2.0.12 only needed an alias for static lib (SDL2::SDL2 to SDL2::SDL2-static, for hunter backward compatibility), same as 2.0.16 (2.0.14 cmake was broken a lot, so I didn't bother). They finally put the exact same alias in SDL2Config.cmake in 2.0.18, so at least this version can be used from upstream.

Reply:
That's awesome! Thank you!